pact/coin-contract/v4/coin-v4.repl

```diff
@@ -509,6 +516,10 @@
 (env-hash (hash "burn-create"))
 (env-data {"doug": ["keys2"]})
 
+; N.B.: this cap is managed via requested <= managed, and so zero or
+; negative quantities may pass the cap and continue failing as normal
```

Comment:
> negative quantities

Hmm? Isn't this enforced in the cap `(enforce (> amount 0.0)...`

Reply:
Yeah - stale comment from previous commits.